src/views/modal_handler.rs

```diff
@@ -205,7 +205,7 @@ impl DisplayModal {
                                 .on_press(Message::ModalHandle(ModalMessage::HideModal))
                                 .style(MODAL_CONNECT_BUTTON_STYLE.get_button_style()),
                         )
-                        .spacing(215);
+                        .spacing(250);
```

Comment:
doesn't this *increase* the space between buttons, leaving less space for the text, not more (or is there something I am not understanding?)

Reply:
This what could have ADDED the problem on this dialog on Pi, as I don't think it was there before...

Reply:
I'd maybe consider more flexible ways to construct this row but keep it looking good.
- I think it would be good if left button was flush with left edge, and right hand button was flush with right edge
- each button big enough for it's text. maybe bigger, no hard
- the buttons could take up most of the width, with just some small white space between them....expanding if the dialog is bigger for some other reason.
- maybe add an empty space in the row between them to keep them from touching (or use padding, not spacing for that?)?

Reply:
The spacing was already there

Reply:
I tried to increase it coz text size reduced to 14 px